main.tf

```diff
@@ -61,9 +61,14 @@
  * ```
  */
 
+data "aws_ecs_cluster" "main" {
```

Comment:
Shouldn't this be `resource "aws_ecs_cluster" "main" {`? Or am I missing something here?

Reply:
This module doesn't create the ECS Cluster. Instead its up to the user to create the ECS cluster and then pass that information to this module where the services are created. So using the `data` instead of `resource` just allows me to make a GET request to AWS and retrieve information like the ARN for use in my module.

Reply:
Aha, okay, that makes sense. Thanks!